tests/ltp/buildenv.sh

```diff
@@ -5,6 +5,7 @@ test_directory=$2
 
 LTP_GIT_TAG="20190930"
 FORK_DISABLE_PATCH="/ltp_fork_disable.patch"
+FIX_SETREUID_PATCH="/fix_setreuid_setreuid07.patch"
```

Comment:
This will not work since you are using same name for 04 and 07 patches. Please update both PRs and use FIX_SETREUID_PATCH04 and FIX_SETREUID_PATCH07 instead.